akka-testkit-typed/src/main/scala/akka/testkit/typed/internal/EffectfulActorContext.scala

```diff
@@ -17,30 +17,31 @@ import scala.concurrent.duration.{ Duration, FiniteDuration }
 /**
  * INTERNAL API
  */
-@InternalApi private[akka] final class EffectfulActorContext[T](name: String) extends StubbedActorContext[T](name) {
+@InternalApi private[akka] final class EffectfulActorContext[T](path: ActorPath) extends StubbedActorContext[T](path) {
 
   private[akka] val effectQueue = new ConcurrentLinkedQueue[Effect]
 
   override def spawnAnonymous[U](behavior: Behavior[U], props: Props = Props.empty): ActorRef[U] = {
     val ref = super.spawnAnonymous(behavior, props)
-    effectQueue.offer(SpawnedAnonymous(behavior, props))
+    effectQueue.offer(SpawnedAnonymous(behavior, props)(ref))
     ref
   }
 
   override def spawnMessageAdapter[U](f: U ⇒ T): ActorRef[U] = {
     val ref = super.spawnMessageAdapter(f)
-    effectQueue.offer(SpawnedAdapter)
+    effectQueue.offer(SpawnedAdapter()(ref))
```

Comment:
to make the naming consistent with `SpawnedAnonymous` this should be `SpawnedAnanymousAdapter`

Reply:
done